Assets/Scripts/Models/Area/TemperatureDiffusion.cs

```diff
@@ -16,7 +17,7 @@ public class TemperatureDiffusion
 {
     private float[,] diffusion;
     private HashSet<Furniture> sinksAndSources;
-    private bool recomputeOnNextUpdate;
+    private bool recomputeOnNextUpdate = true;
```

Comment:
I don't know if it's applicable in this case, but in case it's useful, there's a function in TimeManager where you can request a function to be ran on the next frame. If it would be useful I'll see about digging up what the function is.